src/styles/animation/_animation.scss

```diff
@@ -0,0 +1,40 @@
+.mc-animation {
+  position: relative;
+
+  &.mc-animation--animating {
+    z-index: 99999;
+  }
+
+  &.mc-animation--zoom {
+    > * {
+      transform: scale(1);
+      transition: transform 0.75s cubic-bezier(0.29, 1.185, 1, 1);
+    }
+
+    &.mc-animation--animating > * {
+      transform: scale(1.1);
+    }
+  }
+
+  &.mc-animation--ken-burns {
+    overflow: hidden;
+
+    > * {
```

Comment:
This might cause issues in the future if we have multiple direct children of the `mc-animation` parent (it will animate each child individually instead of the parent)

Reply:
For reference for everybody else, Pontes and I cleared this up, that the component only accepts a single child, and that we could potentially add a wrapper later if we find it's necessary.